### DISCUSSION TITLE
Not clear flow for signin in VS Cide

### DISCUSSION THREAD

**Discussion by Alex2357** [gitkraken/vscode-gitlens#2508] [Q&A]
After logged in it opened web page which says GitLens successfully authenticated with your account!

I copy authorization URL. Not clear where do I past it in vscode as it can be pasted into many places, an opened editor into a some settings value, into Ctrl+Shift+P command. Into terminal. So could it be updated to be clear where do I need to paste that url?

**Comment by jkelroy** [comment]
Hello there, 

When you sign into GitLens, you should see a prompt in the top middle of Visual Studio Code for the authorization URL.

<img width="738" alt="Screen Shot 2023-02-16 at 7 12 18 AM" src="https://user-images.githubusercontent.com/27739784/219388529-c3c9e55e-13d1-4a81-ae89-b3436d3c23e0.png">

Alternatively, you can select `Return to VS Code` to automatically provide this URL to GitLens.

<img width="1006" alt="Screen Shot 2023-02-16 at 7 20 33 AM" src="https://user-images.githubusercontent.com/27739784/219390416-40566efe-c346-46c8-bd5b-55fb821f7425.png">

Does this help?
